examples/source/1.components/amp-story-interactive-poll/index.html

```diff
@@ -0,0 +1,200 @@
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <!-- To begin, import `amp-story-interactive` along your other stories imports. -->
+    <script async custom-element="amp-story-interactive" src="https://cdn.ampproject.org/v0/amp-story-interactive-0.1.js"></script>
+    <style amp-custom>
+    .center {
+      position: absolute;
+      left: 50%;
+      top: 50%;
+      transform: translate(-50%, -50%);
+      width: 22em;
+      font-size: 0.23em;
+    }
+
+    #poll-1 {
+      --interactive-prompt-text-color: rgb(230, 110, 19);
+    }
+
+    #poll-2 {
+      --interactive-accent-color: rgb(9, 63, 4);
+    }
+
+    #poll-3 {
+        --interactive-prompt-background: linear-gradient(120deg, dodgerblue, blue);
+      }
+    </style>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <link rel="canonical" href=".">
+    <title>Interactive polls with results</title>
+  </head>
+  <body>
+    <amp-story standalone
+      publisher="Stories Format"
+      title="Name that artist"
+      poster-portrait-src="./last_supper.jpg"
+      publisher-logo-src="https://amp.dev/static/img/icons/icon-512x512.png">
+      <!-- 
+        ## Overview
+
+        We will create an interactive experience that calculates what animal are you
+        
+        In order to link the poll options to each result state, we will use the attribute `option-{1/2/3/4}-results-category`, so pay special attention to it.
+
+        ## Adding the polls
+        
+        In order to add a poll, we have to specify some attributes:
+
+        - `option-{1/2/3/4}-text` will be used to pass the options to the poll
+        - `option-{1/2/3/4}-confetti` will be used to pass the emojis that will explode when the option is selected. Give it a try and select an option!
+        - `prompt-text` and `prompt-size` will determine the content and size of the question. We'll make it large sine the content is short.
+        - `theme` and `chip-style` will change the way the poll looks.
+        - `option-{1/2/3/4}-results-category` is used to calculate the state of the amp-story-interactive-results element, more info below.
```

Comment:
Suggestions below update voice to active.
```suggestion
        - `option-{1/2/3/4}-text` passes the options to the poll.
        - `option-{1/2/3/4}-confetti` passes the emojis that explode when selecting an option. Give it a try by selecting an option!
        - `prompt-text` and `prompt-size` determine the content and size of the question. We'll make it large sine the content is short.
        - `theme` and `chip-style` change the way the poll looks.
        - `option-{1/2/3/4}-results-category` helps calculate the state of the amp-story-interactive-results element, more info below.
```